examples/terrain_tessellated/main.rs

```diff
@@ -127,12 +131,23 @@ impl<R: gfx::Resources> gfx_app::Application<R> for App<R> {
             .collect();
 
         let (vbuf, slice) = factory.create_vertex_buffer_with_slice(&vertex_data, &index_data[..]);
-        let set = factory.create_shader_set_tessellation(
-            &vs.select(backend).unwrap(),
-            &hs.select(backend).unwrap(),
-            &ds.select(backend).unwrap(),
-            &ps.select(backend).unwrap()
-        ).unwrap();
+
+		let set = if let Ok(gs) = gs.select(backend) {
```

Comment:
indentation seems off?

Reply:
Gonna run cargo fmt in a minute. Tidying up now.

Reply:
`cargo fmt` kinda does a mess - indented manually instead.